read_me.md

```diff
@@ -0,0 +1,8 @@
+<!-- • Create a garage class. -->
+<!-- • Create a method that adds cars to your garage class. -->
+<!-- • Create a method that tells me how many total cars are in the garage. -->
+<!-- • Create a method that counts the number of cars of a certain make. -->
+<!-- • Add an attribute to Car to track its color. -->
+<!-- • Add a method to car that allows me to paint it to change its color. -->
+<!-- • Add a method to garage to allow me to paint all cars the same color. -->
+• Add a method to garage to allow me to paint all cars that are older than a certain year the same color.
```

Comment:
Careful here: we said to that we should be able to paint cars `older` than a certain year. Think that your new method paints cars that are newer than that year.